libs/payment/api/src/lib/payment-provider/payrexx-payment-provider.ts

```diff
@@ -183,7 +250,27 @@ export class PayrexxPaymentProvider extends BasePaymentProvider {
     return {
       state,
       paymentID: gateway.referenceId,
-      paymentData: JSON.stringify(gateway)
+      paymentData: JSON.stringify(gateway),
+      customerID: transaction?.preAuthorizationId
+        ? transaction.preAuthorizationId.toString()
+        : undefined
+    }
+  }
+
+  private mapPayrexxEventToPaymentStatus(
```

Comment:
these kind of private helper methods should be at the top IMO, before the actual logic

Reply:
So this is private method in the class, Im used to add private methods below public ones.
I agree helper functions in front of file.
Any thoughts @Itrulia @elias-summermatter?